mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapper.java

```diff
@@ -92,7 +92,7 @@ public Mailbox findMailboxByPath(MailboxPath path) throws MailboxException {
                 .thenCompose(cassandraIdOptional ->
                     cassandraIdOptional
                         .map(CassandraMailboxPathDAO.CassandraIdAndPath::getCassandraId)
-                        .map(mailboxDAO::retrieveMailbox)
+                        .map(this::retrieveMailbox)
```

Comment:
Explain me this change?

Reply:
Ah.... Got it! Using findMailboxByPatn the acls were not added, right?

Maybe that would deserve a test and a separated commit?

Reply:
Still no test?